### PR TITLE
Web Animations: Add test for Document::getAnimations across iframes

### PR DESCRIPTION
**Pull request by stephenmcgruer**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26805)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/361856722)

**Comment by birtles** [comment]
Looks good, but I'm curious if we actually need the separate iframe document? Elsewhere we just create an empty `<iframe>`:

* web-animations/timing-model/timelines/timelines.html
* web-animations/interfaces/Animation/constructor.html

Or just use a locally defined empty `<iframe>`:

* web-animations/interfaces/Animatable/animate.html
* web-animations/interfaces/Document/timeline.html

File: web-animations/interfaces/Document/getAnimations.html [hunk 1]
```diff
@@ -51,5 +51,28 @@
...
+  const iframe = document.createElement('iframe');
+  iframe.src = '../../resources/iframe.html';
+  document.body.appendChild(iframe);
+  t.add_cleanup(function() { document.body.removeChild(iframe); });
+  iframe.addEventListener("load", t.step_func_done(function() {
+    const iframeTarget = iframe.contentDocument.getElementById("target");
+    const effect = new KeyframeEffect(
+        iframeTarget,
+        [ { backgroundColor: 'green' }, { backgroundColor: 'red' } ],
+        10000);
```
Comment (birtles) [inline web-animations/interfaces/Document/getAnimations.html]: 10s is pretty short -- we should probably use `100 * MS_PER_SEC`.

File: web-animations/interfaces/Document/getAnimations.html [hunk 2]
```diff
@@ -51,5 +51,28 @@
...
+async_test(t => {
+  const iframe = document.createElement('iframe');
+  iframe.src = '../../resources/iframe.html';
+  document.body.appendChild(iframe);
+  t.add_cleanup(function() { document.body.removeChild(iframe); });
+  iframe.addEventListener("load", t.step_func_done(function() {
+    const iframeTarget = iframe.contentDocument.getElementById("target");
+    const effect = new KeyframeEffect(
+        iframeTarget,
+        [ { backgroundColor: 'green' }, { backgroundColor: 'red' } ],
```
Comment (birtles) [inline web-animations/interfaces/Document/getAnimations.html]: (Nit: I'm not sure we actually need any keyframes here. In the interest of keeping this test minimal perhaps we should drop them?)

**Review by ahmadmiz — APPROVED** [review]

**Review by birtles — CHANGES_REQUESTED** [review]
We might decide to drop the extra iframe document if it's not necessary, but at very least we should fix the 10s duration.

**Review by stephenmcgruer — COMMENTED** [review]
Removed the iframe, addressed the other comments.

**Review by birtles — APPROVED** [review]
Thanks!

**Comment by hiikezoe** [comment]
I am wondering we don't normally squash this kind of modifications into the first commit?

**Comment by birtles** [comment]
I probably just hit the wrong button when I merged it.

**Pull request merged**